tigerpath/majors_and_certificates/majors/ARC_2018.json

```
@@ -0,0 +1,201 @@
+{
+  "type": "Major",
+  "name": "Architecture",
+  "code": "ARC",
+  "degree": "AB",
+  "year": 2018,
+  "urls": [
+    "https://ua.princeton.edu/academic-units/school-architecture",
+    "http://soa.princeton.edu/"

```

I would be a bit more specific
```suggestion
    "https://soa.princeton.edu/content/undergraduate-program-architecture",
    "https://soa.princeton.edu/sites/default/files/student%20handbook_undergrad%20F2019-DC.pdf"
```